src/main/app/src/Components/LevelChipComponent/LevelChip.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../Ressources/styling/colors.scss';
+@import '../../Ressources/styling/font.scss';
+
+div.levelChip {
```

Comment:
div kann weg oder haben wir noch ein anderen html tag der den gleichen classname nutzt?

Reply:
It's because of css rule precedence. The original Material UI rule has priority if I just use ".levelChip". Do you know how to fix that?

Reply:
mhmm idk, did you try to import material ui components first and .scss after? but it doesn't really matter